aws-sagemaker-hosted-scorer-cpp/README.md

```diff
@@ -0,0 +1,93 @@
+# DAI Deployment Template for Sagemaker Hosted C++ Scorer
+
+## Overview
+
+### Build Image
+
+Run the following command to build the docker image.
+
+```bash
+docker build -t <aws_account_id>.dkr.ecr.<region>.amazonaws.com/h2oai/sagemaker-hosted-scorer:<tag> .
+```
+
+Verify that the Docker image was created, and take note of the version created.
+
+```bash
+docker images --format "{{.Repository}} \t {{.Tag}}" | grep "h2oai/sagemaker-hosted-scorer"
+```
+
+### Optional: Test the build
+
+After building, run to test the produced Docker container locally like this:
+
+Step 1:  Put a pipeline.mojo and valid license.sig into this directory (aws-sagemaker-hosted-scorer-cpp).
+
+Step 2:  Start the docker instance.
+
+
+```
+docker run \
+    --rm \
+    --init \
+    -ti \
+    -v `pwd`:/opt/ml/model \
+    -p 8080:8080 \
+    harbor.h2o.ai/opsh2oai/h2oai/sagemaker-hosted-scorer:<tag>
+```
+Step 3:  Use the following curl command to test the container locally:
+
+```
+curl \
+    -X POST \
+    -H "Content-Type: application/json" \
+    -d @payload.json http://localhost:8080/invocations
+```
+
+payload.json:
```

Comment:
Is this payload compatible with swagger payload? if yes, we should mention it in documentation.